optimize/backend/src/test/java/io/camunda/optimize/service/util/DatabaseHelperTest.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.optimize.service.util;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import io.camunda.optimize.dto.optimize.DataImportSourceType;
+import io.camunda.optimize.dto.optimize.datasource.DataSourceDto;
+import io.camunda.optimize.dto.optimize.datasource.EngineDataSourceDto;
+import io.camunda.optimize.dto.optimize.datasource.ZeebeDataSourceDto;
+import org.junit.jupiter.api.Test;
+
+public class DatabaseHelperTest {
+
+  public static final String ALIAS = "alias";
+
+  @Test
+  void shouldConstructKeyWithDatabaseTypeAndEngineAlias() {
+    // Given
```

Comment:
```suggestion
    // given
```
Nitpick, but please do this everywhere to align with convention

Reply:
fixed